modules/apps/object/object-web/src/main/resources/META-INF/resources/object_definitions/object_definition/details.jsp

```diff
@@ -24,7 +31,7 @@ renderResponse.setTitle(LanguageUtil.format(request, "edit-x", objectDefinition.
 		module="{EditObjectDetails} from object-web"
 		props='<%=
 			HashMapBuilder.<String, Object>put(
-				"backURL", ParamUtil.getString(request, "backURL", String.valueOf(renderResponse.createRenderURL()))
+				"backURL", ParamUtil.getString(request, "backURL", backURL)
```

Comment:
@igor-franca Here you can just put the backURL parameter, because you already did the ParamUtil in row 15.
Change it and test, please.

Reply:
Thanks for noticing, see my changes [here](https://github.com/liferay-objects/liferay-portal/compare/4622609649b111c3b6df5fb4cfc079be97d57204..9f4bbf75a71ba8e479a1cb9e13bf79756513c39c)